apps/src/storage/dataBrowser/dataUtils.js

```diff
@@ -19,7 +18,7 @@ export const ChartType = {
   CROSS_TAB: 4
 };
 
-export function getDatasetInfo(tableName) {
+export function getDatasetInfo(tables = [], tableName) {
```

Comment:
Move the default param (i.e. `tables = []`) to after tableName. i.e. with the way it's written right now, if someone calls `getDatasetInfo('cats')`, you'll get a TypeError. Whereas, I think the behavior you are going for is for it to return `undefined`.